docs/custom_conf.py

```diff
@@ -141,6 +141,14 @@
 # Addons-by-default, see: https://about.readthedocs.com/blog/2024/07/addons-by-default/
 if os.environ.get('READTHEDOCS', '') == 'True':
     html_context['READTHEDOCS'] = True
+    # The following are needed, see: https://github.com/pradyunsg/furo/blob/main/docs/conf.py#L135.
+    html_context['current_version'] = 'latest'
```

Comment:
I'm not certain about this. We never used to see a banner warning about "latest" and I don't think we want one. So maybe we want "stable" to be the current version?

Or would this remove that banner because it's telling rtd that "latest" is (essentially) "stable"? If so then it would be great indeed.

Reply:
my 2c: let's merge it and see

Reply:
My main concern: I don't want all our "/latest" docs URLs to break or go stale.